lib/mincer/manifest.js

```diff
@@ -176,6 +176,7 @@ function gzip(data) {
  *  - `embedMappingComments` (false) - set `true` to embed sourcemap url
  *    into created files
  *  - `compress` (false) - set `true` to also create gzipped files
+ *  - `preventXSSI` (true) - set `false` to skip adding XSSI protection header
```

Comment:
IMHO active inverse flags are dangerous. I'd suggest something like `noSourceMapProtection` (may be need clarification).